### ISSUE TITLE
How to add :cache 500 to my ~/.config/evcxr/init.evcxr

### ISSUE THREAD

**Issue by aksjfds** [evcxr/evcxr#362]
I didn't find `init.evcxr` in my pc, should i create this file and then add `:cache` 500 to it?

**Comment by davidlattimore** [comment]
Yep, evcxr doesn't automatically create it for you - although that's an interesting idea that I hadn't thought of

**Comment by aksjfds** [comment]
> Yep, evcxr doesn't automatically create it for you - although that's an interesting idea that I hadn't thought of

how can I use `:cache`?

**Comment by davidlattimore** [comment]
You should just need to create a file `~/.config/evcxr/init.evcxr` containing `:cache 500`. That's assuming you're on Linux. If you're on a different platform, the filename will be different. If you're not sure what the path to the config file should be for your platform, you can run (from evcxr):
```
:dep dirs
dirs::config_dir().unwrap().join("evcxr").join("init.evcxr")
```
